tests/conftest.py

```diff
@@ -9,17 +10,13 @@ def debug(request):
     return request.param
 
 
-@pytest.fixture
+@pytest.fixture(autouse=True)
 def loop(event_loop, debug):
     event_loop.set_debug(debug)
     return event_loop
 
 
-@pytest.fixture
-def session(loop):
-    async def create_session(loop):
-        return aiohttp.ClientSession()
-
-    session = loop.run_until_complete(create_session(loop))
-    yield session
-    loop.run_until_complete(session.close())
+@pytest_asyncio.fixture
```

Comment:
I think even this can be removed, and a regular pytest.fixture used, even when it's async.

Reply:
Sure, it's been needed before you set 'auto' in pytest asyncio :)